action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     description: "Seconds to wait between Checks API requests"
     required: false
     default: "10"
+  api-endpoint:
+    description: "Github API Endpoint to use."
+    required: false
+    default: ""
```

Comment:
To double down on @codezninja 's comment, could you try with `null` here to avoid the `.strip.empty?` and use the `ENV.fetch` with default syntax?

Reply:
Will do. I'll test it out and update the rp

Reply:
so `null` in action.yml didn't work but the sugggestion from @matiasalbarello worked without the need for strip. So I've updated the entrypoint with it.

Reply:
This is what I get when I try to set null 
<img width="1358" alt="image" src="https://user-images.githubusercontent.com/4400773/184291939-07dca536-48f0-45ab-85fb-e89026c9834d.png">